source/common/logger/ur_sinks.hpp

```diff
@@ -195,6 +197,35 @@ class FileSink : public Sink {
     std::ofstream ofstream;
 };
 
+class CallbackSink : public Sink {
+  public:
+    CallbackSink(std::string logger_name, bool skip_prefix = false,
```

Comment:
I can't find where the `logger_name` is used for. I realize it's part of the `Sink` interface, but it seems to be unused?

Reply:
it gets used here in the `log` method from the generic `Sink` https://github.com/oneapi-src/unified-runtime/blob/main/source/common/logger/ur_sinks.hpp#L26 but that doesn't get used in this new class